nemo_skills/dataset/include/prepare.py

```
@@ -0,0 +1,91 @@
+        "extract_regex": build_extract_regex(language),
+        "relaxed": False,
+        "subset_for_metrics": language,
+        "category": category,
+        **get_mcq_fields(target_question, target_options, language, subject),
+        **copy_other_fields(entry),
+    }
+
+
+def collect_entries(languages, datasets):
+    entries = []
+    for dataset, lang in zip(datasets, languages):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)